src/lndk_offers.rs

```diff
@@ -350,7 +372,11 @@ pub async fn validate_amount(
 
 pub async fn get_destination(offer: &Offer) -> Destination {
     if offer.paths().is_empty() {
-        Destination::Node(offer.signing_pubkey())
+        if let Some(signing_pubkey) = offer.signing_pubkey() {
+            Destination::Node(signing_pubkey)
+        } else {
+            Destination::BlindedPath(offer.paths()[0].clone())
```

Comment:
if we're in the offers.paths().is_empty() branch, offer.paths()[0] doesn't exist?